docs/users/installation.md

```diff
@@ -0,0 +1,225 @@
+---
+id: installation
+title: Installation
+---
+
+To run scalafix on your project, you must first install the Scalafix sbt plugin
+or command line interface. Currently, Scalafix does not provide any IDE
+integrations.
+
+| Scalafix Version | Scalameta(SemanticDB) Version | Scala Version           |
+| ---------------- | ----------------------------- | ----------------------- |
+| 0.5.10           | 2.1.7                         | 2.11.12 / 2.12.4        |
+| @VERSION@        | @SCALAMETA@                   | @SCALA211@ / @SCALA212@ |
```

Comment:
This is pain point number one. I would prefer we release a Scalameta version for each Scala Version. I think we discussed this offline and you said it would be to much work. We should make it really clear here what is supported.